lib/screen/book_editor.dart

```diff
@@ -36,6 +42,7 @@ class _BookEditorState extends State<BookEditor> {
 
   @override
   Widget build(BuildContext context) {
+    final uid = Provider.of<CurrentUser>(context).data.uid;
```

Comment:
So if I understand you're using the user's uid as the user's node in the book database, is that right?

Reply:
Yes exactly like it was done with the note node